docs/docsite/rst/playbooks_tags.rst

```diff
@@ -73,7 +81,11 @@ Example::
 Tag Inheritance
 ```````````````
 
-You can apply tags to more than tasks, but they ONLY affect the tasks themselves. Applying tags anywhere else is just a convenience so you don't have to write it on every task::
+When tagging items other than dynamic inclusions (see 'include_role' and
+'include_tasks'), you can apply ``tags:`` to structures above tasks. When
+Ansible processes these, ONLY the tasks they contain are tagged. Applying tags
+anywhere other than tasks is just a convenience so you don't have to tag all
+indivdual tasks. This example tags all tasks in a play::
```

Comment:
not sure i understand what you are saying here

Reply:
I did reword both these sections, and also tried a top to bottom review of the page to try to introduce concepts more slowly. As written, some pieces were being introduced ahead of their sections, making it difficult to describe simpler examples first.